File: 2018-12-23/2577/2577.c
#include <stdio.h>
#include <string.h>
#define _CRT_SECURE_NO_WARNINGS 
int main(){
	int arr[10] = {0,};
	int A, B, C;
	int oneNum=0, startNum;
	int lastNum=0;
	scanf("%d\n%d\n%d", &A, &B, &C);
	int all = A*B*C;
	for(int i = 100000000; i>=1; i/=10){
		if(all>=i){
			startNum = i;
			break;
		}
	}

	for(int i = startNum; i>=1; i/=10){
			oneNum = all/i;
			arr[oneNum] +=1;
			all-=oneNum*i;
	}

	for(int i =0;i<10;i++)
		printf("%d\n", arr[i]);
	
}

File: 2018-12-23/2577/2577.py
import sys

A = int(sys.stdin.readline())
B = int(sys.stdin.readline())
C = int(sys.stdin.readline())

arr = [0 for i in range(0,10)]

all = A*B*C

startNum = 100000000
for i in range(0, 3):
	if(all >=  startNum):
		break
	startNum//=10;


while(startNum !=1):
	index = all//startNum
	arr[index]+=1
	if(startNum==10):
		arr[all%10]+=1
	all -= index*startNum
	startNum//=10

for i in range(0,10):
	print(arr[i])	

